src/test/java/pages/LoginPage.java

```diff
@@ -0,0 +1,29 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class LoginPage extends BasePage{
+
+    //locators
+    static By emailLocator = By.cssSelector(("[type = 'email']"));
+    static By passwordLocator = By.cssSelector("[type = 'password']");
+    static By submitLocator = By.cssSelector("button[type = 'submit']");
```

Comment:
Thank you. I meant to write "private static". I know that by default the variables and methods in java are private, so over here I have been just lazy to type in the word private for those particular variables, which is not a good habit, apologies, just a habit of knowing and relying on defaults, I know that you can't do it everywhere, in C++ there are almost no defaults. 
I need however static since I'm using those variables inside static helper methods.